src/tempo.py

```diff
@@ -73,16 +74,17 @@ def config(
         Only activate the provided receivers.
         """
 
+        external_hostname = re.sub(r"^https?:\/\/", "", coordinator._external_url)
```

Comment:
is this something we should expose in the coordinator? if so, add a TODO here and open an issue on coslib

Reply:
Its related to ingress so I kept it at the charm level. Added a property in the charm code and passed it to Tempo object to be used instead of defining it here